examples/enclave_manager/setup.py

```diff
@@ -61,57 +57,34 @@
 
 include_dirs = [
     module_src_path,
+    enclave_bridge_path,
     os.path.join(tcf_root_dir, 'tc/sgx/common'),
     os.path.join(tcf_root_dir, 'tc/sgx/common/crypto'),
     os.path.join(module_src_path, 'build'),
-    os.path.join(sgx_sdk_env,  'include'),
     os.path.join(tcf_root_dir, 'tc/sgx/common/packages/base64')
 ]
 
 library_dirs = [
+    os.path.join(enclave_bridge_path, 'build/lib'),
     os.path.join(tcf_root_dir, "tc/sgx/common/build"),
-    os.path.join(os.environ['SGX_SDK'], 'lib64'),
     os.path.join(sgx_ssl_env, 'lib64'),
-    os.path.join(sgx_ssl_env, 'lib64', 'release')
 ]
 
 libraries = [
-    'uavalon-common',
-    'uavalon-crypto',
+    'uavalon-parson',
```

Comment:
Can we avoid parson dependency in untrusted part

Reply:
parson library is needed because the bridge wrapper files use json functions offered by parson. 
I will raise another PR to remove parson dependency on untrusted cpp code. Then we can revisit and remove parson dependency here.